src/Client/Web/Bit.Client.Web.BlazorUI/Styles/Fluent/Notifications/bit-message-bar.fluent.scss

```diff
@@ -11,34 +11,88 @@
     display: flex;
     word-break: break-word;
     min-height: 32px;
+    align-items: flex-start;
+
+    .text-container {
+        flex-grow: 1;
+        display: -webkit-box;
+        -webkit-box-orient: vertical;
+        overflow: hidden;
+        -webkit-line-clamp: 1;
+        margin: 8px 0px 8px 12px;
+        padding: 0;
+        text-align: left;
+    }
+
+    .not-truncate {
+        -webkit-line-clamp: unset;
```

Comment:
do not use "-webkit-line-clamp" to truncate the text. see the Fluent UI sample to get an idea about the correct way of implementing it using overflow and text-overflow properties.

Reply:
Fixed